test/transmuxer.test.js

```diff
@@ -2074,6 +2076,45 @@ QUnit.test('calculates baseMediaDecodeTime values from the first DTS ever seen a
   QUnit.equal(tfdt.baseMediaDecodeTime, 90, 'calculated baseMediaDecodeTime');
 });
 
+QUnit.test('doesn\'t adjust baseMediaDecodeType when configured to', function() {
```

Comment:
Minor: s/baseMediaDecodeType/baseMediaDecodeTime/, and maybe specify "when configured to keep original timestamps"

Reply:
Done